internal/store/fleet.go

```diff
@@ -138,17 +138,17 @@ func (s *FleetStore) Get(ctx context.Context, orgId uuid.UUID, name string, opts
 		return nil, flterrors.ErrResourceNotFound
 	}
 
-	summary := api.DevicesSummary{
-		Total: fleet.DeviceCount,
-	}
+	var summary *api.DevicesSummary
```

Comment:
so is the expectation that summarty is nil when passed to model.WithDevicesSummary? we are certain its handled the rest of the way? in general I think wouldn't it be stafter to initialize an owned summary then pass reference like it was before. perhaps a short note would clear that up so I don't break it next week :)

Reply:
just read the pr comment i think that summarizes this well, just add that note here.